config_example.json

```diff
@@ -135,6 +135,8 @@
    "httpTimeout": 15000000000,
    "httpUserAgent": "",
    "authenticatedApiSupport": false,
+   "apiWithdrawalSupport": true,
```

Comment:
This seems non-needed as the withdraw permissions are set in the exchanges SetDefaults() without any config handling. In the future it would be nice to open the ability to adjust via config.

Reply:
Whoops, old code >_>